modulemd/tests/test-valgrind.py

```diff
@@ -41,8 +47,10 @@ def exec_valgrind(test):
             "/usr/bin/valgrind "
             "--leak-check=full "
             "--suppressions=/usr/share/glib-2.0/valgrind/glib.supp "
+            "--suppressions=%s "
+            "--gen-suppressions=all "
```

Comment:
I don't think we want to be generating suppressions for every run.

Reply:
> I don't think we want to be generating suppressions for every run.

I don't think there's much additional overhead for valgrind, and the XML only gets dumped to the logfile when one of the fatal error types is identified by `test-valgrind.py`. (Which, by the way, is a bug I just fixed. Previously, once any of the valgrind tests failed, the XML for _all_ remaining tests was dumped to the log!) Nevertheless, I removed the line to generate the suppressions.